tests/test_donor.py

```diff
@@ -145,6 +145,7 @@ def test_unignore_not_ignored(self, user, testapp):
 class TestOverride:
     @pytest.mark.parametrize("rodne_cislo", sample_of_rc(5))
     def test_override(self, user, testapp, rodne_cislo):
+        skip_if_ignored(rodne_cislo)
```

Comment:
Souvisí toto nějak s opravou velkých písmen u vstupů? Pokud ne, mělo by to být ve vlastní žádosti.

Reply:
Je to v samostatném commitu a někdy bez tohoto padají testy, takže to komplikuje PR, proto takto.